### ISSUE TITLE
Breadcrumbs should be navigatible with the keyboard

### ISSUE THREAD

**Issue by chrisrrowland** [tl-its-umich-edu/canvas-course-manager-next#73]
I noticed that pressing spacebar while a breadcrumb link is selected does not follow the link.
I'm assuming this should work, though neither Axe or Lighthouse analysis complained about the current state.

**Comment by gsilver** [comment]
Nor sure this was ever a problem @chrisrrowland - links are activated with the "Enter" key.
